articles/other/other-how-view-status-info.md

```diff
@@ -0,0 +1,121 @@
+
+### Service status
+
+As well as the overall status of the platform, the Service Status page also shows the status of each service on the UKCloud platform.
+
+For each service, you can see the availability over the last 90 days, with each day represented by a bar. Hover over a bar to see a popup with details for that day. If there are no issues with a service, the status is reported as `Operational`.
+
+![Service status](images/other-status-service.png)
+
+The uptime figure indicates what percentage of the last 90 days the service was available for.
+
+Click the **+** icon to expand the service to see the status of specific components within the service. For example, for Connectivity, you can see the status of each connectivity type (Internet, PSN, HSCN, Janet, RLI); for UKCloud for VMware, you can see the status for each UKCloud zone.
+
+If service availability is impacted, the bar for that day displays in a different colour and the status is reported as `Under Maintenance`, `Degraded Performance` or `Outage`.
+
+![Service with degraded performance](images/other-status-outage.png)
+
+## Subscribing to the Service Status page
+
+You can subscribe to the Service Status page to receive updates when information is updated.
+
+1. On the UKCloud Service Status page, click the **Subscribe to Updates** button.
+
+    ![Subscribe to Updates button](images/other-status-btn-subscribe.png)
+
+2. To receive email notifications:
+
+    1. Click the envelope icon, enter your email address, then click **Subscribe via Email**.
+
+        ![Subscribe via Email](images/other-status-notification-email.png)
+
+    2. Confirm your subscription by clicking the **Confirm subscription** button in the email you receive.
+
+    3. On the *Notifications Subscription* page, you can specify individual components for which you want to receive notifications.
+
+        ![Notifications Subscription page](images/other-status-subscriptions.png)
+
+3. To receive text notifications:
+
+    1. Click the telephone handset icon, enter your phone number, then click **Subscribe via Text Message**.
+
+        ![Subscribe via Text Message](images/other-status-notification-text.png)
+
+    2. Click the link in the confirmation text message you receive to manage your subscriptions.
+
+4. To receive webhook notifications, click the code icon (**<>**), enter the URL of the endpoint to send the webhook to and the email address to send any associated error messages, then click **Subscribe to Notifications**.
+
+    ![Subscribe via Webhook](images/other-status-notification-webhook.png)
+
+    You'll need to set up an endpoint to receive the webhook requests. For more information about webhooks, see the Atlassian Statuspage documentation: [Webhook Notifications](https://help.statuspage.io/help/webhook-notifications). For example code for setting up an endpoint, see [*How to use webhooks to receive service status notifications*](other-how-use-webhooks.md).
```

Comment:
all one word?

Reply:
Statuspage is the name of the product from Atlassian -- all one word